Includes/subAppRouter.cpp

```diff
@@ -98,16 +190,14 @@ void SubAppRouter::handleButtonDown(const SDL_ControllerButtonEvent &event) {
     case SDL_CONTROLLER_BUTTON_START:
       app.onStartPressed();
       break;
+    default:outputLine("Ignoring invalid button ID %d\n", button);
```

Comment:
Please insert a line break between default: and what follows.

Reply:
Wow, that's a weird autoformat artifact. Fixed.